plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -188,33 +188,30 @@ public void testTableNameClash()
 
     @Test
     public void testDropSchema()
+            throws Exception
     {
         String schema = "Test_Drop_Case_Sensitive";
-        bigQuerySqlExecutor.execute(format("DROP SCHEMA IF EXISTS `%s`", schema));
-        bigQuerySqlExecutor.execute(format("CREATE SCHEMA `%s`", schema));
-
-        assertUpdate("DROP SCHEMA " + schema.toLowerCase(ENGLISH));
+        try (AutoCloseable firstSchema = withSchema(schema);) {
```

Comment:
nit: Rename `schema` -> `schemaName` then you can use `schema` for the actual schema.

Here and elsewhere.